bin/fun-deploy.js

```diff
@@ -22,6 +22,7 @@ program
 
   .option('-t, --template [template]', 'The path of fun template file.')
   .option('-c, --only-config', 'Update only configuration flags')
+  .option('-y, --assume-yes', 'Assume Yes to all queries and do not prompt')
```

Comment:
Automatic yes to prompts. Assume "yes" as answer to all prompts and run non-interactively.

Reply:
done